app/Http/Requests/ItemImportRequest.php

```diff
@@ -38,9 +40,8 @@ public function import(Import $import)
 
         $filename = config('app.private_uploads').'/imports/'.$import->file_path;
         $import->import_type = $this->input('import-type');
-        $class = ucfirst($import->import_type);
-        $classString = "App\\Importer\\{$class}Importer";
```

Comment:
I have *always* felt really uncomfortable with the way this works - I really appreciate that you took the time to fix it! Your way is so much better!